.travis.yml

```diff
@@ -41,6 +44,12 @@ install:
       wget https://mirrors.kernel.org/ubuntu/pool/universe/v/vulkan/libvulkan-dev_1.1.73+dfsg-1_amd64.deb;
       sudo apt-get install --only-upgrade dpkg;
       sudo dpkg -i libglew2.0_2.0.0-5_amd64.deb libglew-dev_2.0.0-5_amd64.deb libvulkan1_1.1.73+dfsg-1_amd64.deb libvulkan-dev_1.1.73+dfsg-1_amd64.deb;
+      if [ "$CXX" = "clang++-6.0" ]; then
+        wget -O - https://apt.llvm.org/llvm-snapshot.gpg.key | sudo apt-key add -;
+        sudo apt-add-repository "deb http://apt.llvm.org/trusty/ llvm-toolchain-trusty-6.0 main";
+        sudo apt-get update;
+        sudo apt-get install -y clang-6.0 lld-6.0;
```

Comment:
Just add clang-6.0 and clang++-6.0 in the packages section, you shouldn't need lld, and rpcs3 doesn't link with lld 6.0 anyway. and add `llvm-toolchain-trusty-6.0` to the *sources* section, right before the *packages* section.